config/runtime.exs

```diff
@@ -17,24 +17,13 @@ import Config
 # Alternatively, you can use `mix phx.gen.release` to generate a `bin/server`
 # script that automatically sets the env var above.
 if System.get_env("PHX_SERVER") do
-  config :phil, PhilWeb.Endpoint, server: true
+  config :phil, PhilWeb.Endpoint, server: true |> IO.inspect(label: "Endpoint config: server")
```

Comment:
Maybe I'm wrong, but does it make sense to generate output in `runtime.exs`? Thinking this would be better handled in our application code and inspected after everything is set up.

Reply:
Maybe this was introduced for development testing :)

Reply:
Removed in [41c977a](https://github.com/mbta/phil/pull/6/commits/41c977ac938380ef1bfc59c5770a3e0b559749a7)